test/extended/util/image_helpers.go

```diff
@@ -97,3 +97,15 @@ func CreateResource(jsonFilePath string, oc *CLI) error {
 	err := oc.Run("create").Args("-f", jsonFilePath).Execute()
 	return err
 }
+
+//NewApp performs an oc new-app invocation from the supplied json file
+func NewApp(jsonFilePath string, oc *CLI) error {
+	err := oc.Run("new-app").Args(jsonFilePath).Execute()
+	return err
+}
+
+//TagImageStream performs and oc tag invocation with the provided parameters
+func TagImageStream(source, destination string, oc *CLI) error {
+	err := oc.Run("tag").Args(source, destination, "--config", KubeConfigPath()).Execute()
+	return err
```

Comment:
in hindsight yes that seem obvious (i think i was originally expecting something more elaborate in those methods) ;-)

getting rid of both